extensions/panache/panache-common/deployment/src/main/java/io/quarkus/panache/common/deployment/visitors/KotlinPanacheClassOperationGenerationVisitor.java

```diff
@@ -537,6 +537,9 @@ public MethodVisitor visitMethod(int access, String name, String descriptor, Str
                 .orElse(null);
         if (methodInfo != null && !methodInfo.hasAnnotation(DOTNAME_GENERATE_BRIDGE)) {
             return super.visitMethod(access, name, descriptor, signature, exceptions);
+        } else if (name.contains("$")) {
```

Comment:
does jacoco copy the annotation as well?  i'd expect the first clause to pass the method through.

Reply:
It's a new method, there is no annotation to copy